landlab/components/spatial_disturbance/tests/test_spatial_disturbance.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
Could you please add better tests of the component?

Reply:
Can you please give me more pointers? What do you mean by better tests?

Reply:
done